spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/server/MissingWebServerFactoryBeanException.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2012-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.web.server;
```

Comment:
I'm not sure where it should go instead, but this package doesn't feel right to me as the failure is specific to the web server being used in an application context. I'm not sure it's worth creating a `org.springframework.boot.web.server.context` just for this, and the two existing `context` packages are Servlet and reactive-specific.

Reply:
The same as above.